src/Matrix.cc

```diff
@@ -1153,6 +1154,36 @@ NAN_METHOD(Matrix::BilateralFilter) {
   info.GetReturnValue().Set(Nan::Null());
 }
 
+NAN_METHOD(Matrix::Sobel) {
```

Comment:
nit; please add a header explaining what this function does. Referencing opencv sobel documentation is good enough in my opinion.